analyzers/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveClassCoupling.cs

```diff
@@ -38,18 +39,16 @@ public sealed class AvoidExcessiveClassCoupling : ParameterLoadingDiagnosticAnal
         internal const string DiagnosticId = "S1200";
         private const string MessageFormat = "Split this {0} into smaller and more specialized ones to reduce its " +
             "dependencies on other classes from {1} to the maximum authorized {2} or less.";
```

Comment:
We need to update also the diagnostic message to not contain only classes.
"dependencies on other **classes** from"